### ISSUE TITLE
dublicate event 'pageChange'

### ISSUE THREAD

**Issue by tyrbonit** [njleonzhang/vue-data-tables#127]
If set property :pagination-def="{pageSize: 20, pageSizes: [20, 50, 100], currentPage: 2}"  in "data-tables-server" and click on page then function in ":load-data " calls 2 times

**Comment by njleonzhang** [comment]
can you provide a reproduce sample? just fork this: https://jsfiddle.net/zpczjl/xboja87c/

**Comment by tyrbonit** [comment]
Yes, of course? see https://jsfiddle.net/tyrbonit/udkcj4m5/1/

**Comment by njleonzhang** [comment]
@tyrbonit tnx for reporting this bug, will fix ASAP

**Comment by njleonzhang** [comment]
@tyrbonit you should not write the object directly in template, which will cause the component always render with the this data. 
```
:pagination-def="{pageSize: 20, pageSizes: [20, 50, 100], currentPage: 2}"
```

```
{pageSize: 20, pageSizes: [20, 50, 100], currentPage: 2} === {pageSize: 20, pageSizes: [20, 50, 100], currentPage: 2} // false
```

https://jsfiddle.net/udkcj4m5/2/

**Comment by tyrbonit** [comment]
Thank you, it works!
I have another problem:
Then init data-tables-server, i request data from server and use :table-props="tableProps" for define sorting parameters, but I got two requests instead of one, because queryInfo does not contain a sortInfo
See https://jsfiddle.net/tyrbonit/udkcj4m5/3/

**Comment by njleonzhang** [comment]
@tyrbonit fix in 3.1.2
